File: JavaLab/src/main/java/ch2/section2/sub5/Quiz2.java
package ch2.section2.sub5;

public class Quiz2 {
    public static void main(String[] args) {
        int month = 8;

        switch (month) {
            case 12:
            case 1:
            case 2:
                System.out.println("겨울입니다.");
                break;
            case 3:
            case 4:
            case 5:
                System.out.println("봄입니다.");
                break;
            case 6:
            case 7:
            case 8:
                System.out.println("여름입니다.");
                break;
            case 9:
            case 10:
            case 11:
                System.out.println("가을입니다.");
                break;
        }
    }
}

File: KotlinLab/KotlinLab/src/main/kotlin/ch2/section4_control/q3/Quiz2.kt
package org.example.ch2.sec4_control.q3

import java.text.SimpleDateFormat
import java.util.Calendar



fun saySeason(inputDate: String): String {
    val date = SimpleDateFormat("yyyy-MM-dd").parse(inputDate)

    val calendar = Calendar.getInstance()
    calendar.time = date

    val month = calendar.get(Calendar.MONTH) + 1

    return when (month) {
        12, 1, 2 -> "겨울이군요"
        3,4,5 -> "봄이군요"
        6,7,8 -> "여름이군요"
        9,10,11 ->  "가을이군요"
        else -> "잘못된 입력이군요"
    }
}

fun main() {
    println(saySeason("2025-04-30"))
    println(saySeason("2025-12-30"))
}